widgetssdk/src/main/java/com/glia/widgets/callvisualizer/EndScreenSharingActivity.kt

```diff
@@ -18,6 +18,7 @@ class EndScreenSharingActivity : AppCompatActivity(), EndScreenSharingView.OnFin
         screenSharingView.onFinishListener = this
 
         val controller = Dependencies.getControllerFactory().endScreenSharingController
+        controller.listenForEngagementEnd()
```

Comment:
Controller should be the one giving out orders based on events, not taking orders as input.

I see two options here:
1. have the controller always listen for engagement end
2. pass view lifecycle events like onResume and onPause to the controller and let it decide to do when that happens

Reply:
@gugalo , @Pelkar Yes, agree, thank you.
Made change 6ce113f07f4c2060a91bd09ae9816bb6b017b3ae.
Let me know if it looks good to you now.